### PR TITLE
Add listing of available addons

### PR DESCRIPTION
**Pull request by rondale-sc**
Originally posted:
http://hashrocket.com/blog/posts/a-compendium-of-hooks-in-embercli

Anchors are above headers explicitly rather than relying on Github to
place them.  This is to allow easier updating to original post down the
road, and doesn't affect the way GitHub will render them.

### CODE REVIEW

**Comment by stefanpenner** [comment]
@joostdevries @rondale-sc i have lost track of this one, is their anything that needs to be done yet?

File: ADDON_HOOKS.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+**Examples:**
+
+- [ember-cli-cordova](https://github.com/poetic/ember-cli-cordova/blob/v0.0.14/index.js#L19)
+
+```js
+  // https://github.com/rwjblue/ember-cli-divshot/blob/v0.1.6/index.js
+  includedCommands: function() {
+    return {
+      'divshot': require('./lib/commands/divshot')
+    }
```
Comment (stefanpenner) [inline ADDON_HOOKS.md]: @rondale-sc ^^

**Comment by Samsinite** [comment]
contentFor hook still needs added :)

File: ADDON_HOOKS.md (hunk 1 above)
Comment (rondale-sc) [inline ADDON_HOOKS.md]: Fixed :)  (sorry took so long)

**Comment by stefanpenner** [comment]
@rondale-sc let me know when its good to merge

**Comment by rondale-sc** [comment]
@stefanpenner The ember-cli links are all hardcoded to v0.1.4.  Which I think is still mostly accurate, but I'm not entirely certain how we can keep those links in sync.

**Comment by rondale-sc** [comment]
@Samsinite contentFor and postProcessTree have been added.  :)  Thanks (sorry it took so long for me to get to it)

**Comment by rondale-sc** [comment]
Should be good to merge.  (not entirely certain what is happenin' with Travis)

:beers: Cheers :beers:

**Comment by stefanpenner** [comment]
@rondale-sc thanks bro!!

**Comment by stefanpenner** [comment]
ya not sure about keeping the links in-sync, sounds manual. Or... ?

**Comment by stefanpenner** [comment]
cc @joostdevries

**Comment by rondale-sc** [comment]
I suspect it'll be manual.  Maybe, a small script on a cron to parse the Changelog for any of the addons (being as fuzzy as possible).  

Seems pretty error prone, that.

**Comment by chadhietala** [comment]
:+1: Awesome job everyone.

**Pull request merged**